python/ray/experimental/dag/py_obj_scanner.py

```diff
@@ -46,8 +46,7 @@ def __init__(self):
         from ray.experimental.dag.function_node import FunctionNode
         from ray.experimental.dag.class_node import ClassNode, ClassMethodNode
         from ray.experimental.dag.input_node import InputNode, InputAtrributeNode
-        from ray.serve.pipeline.deployment_node import DeploymentNode
-        from ray.serve.pipeline.deployment_method_node import DeploymentMethodNode
+        from ray.serve.api import DeploymentNode, DeploymentMethodNode
```

Comment:
we really shouldn't have any references to library APIs here if possible, is this necessary or can we refactor to remove it?